magento/magento2
# Remove configuration options from useless scopes

### Description
These four settings are advertised as being able to be set at `store scope`, however there is no sense in trying to set these at this level. Also, the relevant section isn't visible at those scopes anyway.
- Admin Account Sharing
- Password Reset Protection Type
- Max Number of Password Reset Requests
- Min Time Between Password Reset Requests

![Screenshot_2022-04-22_19-21-42](https://user-images.githubusercontent.com/334786/164772990-7d471907-e3d2-4e07-870e-aca2f4f360ed.png)

### Manual testing scenarios
1. Navigate in the admin to Stores -> Settings -> Configuration -> Advanced -> Admin -> Security
1. Notice the scope hint next to each option. This is `[global]` for most, but `[store view]` for others.
1. Change the scope to something non-global.
1. Notice that the configuration path no longer exists / isn't visible in the admin.
1. Apply the changes in this pull request.
1. Navigate in the admin to Stores -> Settings -> Configuration -> Advanced -> Admin -> Security
1. Notice the scope hint next to each option. This is now `[global]` for all.

### Contribution checklist
